src/io/files-util.cpp

```diff
@@ -221,6 +221,7 @@ static errr counts_seek(PlayerType *player_ptr, int fd, uint32_t where, bool fla
 {
     char temp1[128], temp2[128];
     auto short_pclass = enum2i(player_ptr->pclass);
+    (void)memset(temp1, 0, sizeof(temp1))
```

Comment:
The semicolon is missing, which causes an error in the CI check, but it is recommended to declare the variable as `temp1[128]{}` instead of using memset if you want to zero-clear the variable (Strictly speaking, this is not necessary, but it is a good idea to do the same for `temp2` as a side note.).